verifier/src/identity.rs

```diff
@@ -713,6 +717,28 @@ mod test {
         assert_eq!(format!("\n{displayable}"), textwrap::dedent(expected));
     }
 
+    #[test]
+    fn identities_verifier_no_identities() {
```

Comment:
:facepalm: the "empty" test case is normally one of the test cases to always have for a sequence of items

Reply:
better late than never 😄